core/rend/gles/gltex.cpp

```diff
@@ -260,8 +273,12 @@ struct TextureCacheData
 		}
 	}
 
-	void Update()
+	void Update(bool isSourceVram = true, u16 *sourceData = NULL)
```

Comment:
Writing directly to vram and avoiding `FBT::texData` would make  `isSourceVram` and `sourceData` wouldn't it?

Reply:
It is added for the same reason as above -> to improve performance RTT is implemented in a way that vram is omitted (because vram is not valid in the next frame), why is that I wrote in this comment (third paragraph): https://github.com/reicast/reicast-emulator/pull/1460#issuecomment-439205071